.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "5.10.1"
```

Comment:
the server is on 4.2.0 You may want to test that only (since the client makes sure the server tests pass)

Reply:
changed to 4.2.0